packages/nuxt-plugin/vitest.config.ts

```diff
@@ -18,6 +18,5 @@ export default defineProject({
       }),
       enforce: "pre",
     },
-    ...plugins,
```

Comment:
I see we remove these `...plugins` in the vitest.config.ts files now - what's the impact/purpose of that just for my understanding?

Reply:
There was only one plugin being used ts config path, which i removed because it was causing issues with the example apps while trying to run the test runners. The only impact is that we import things in the core tests slightly differently with relative imports rather than a `@` for the test utils.